src/transformers/integrations/flex_attention.py

```diff
@@ -213,23 +230,44 @@ def flex_attention_forward(
     scaling: Optional[float] = None,
     softcap: Optional[float] = None,
     head_mask: Optional[torch.Tensor] = None,
+    output_attentions: bool = False,
+    dropout: float = 0.0,
+    eager_fallback: Optional[Callable] = None,
     **kwargs,
 ) -> Tuple[torch.Tensor, torch.Tensor]:
+    if output_attentions or head_mask is not None or dropout > 0:
+        logger.warning_once(
+            "Falling back to eager attention because `flex_attention` does not support"
+            " `output_attentions=True`, `head_mask`, or `dropout`."
+        )
+        return eager_fallback(
```

Comment:
same here! And head mask is something we kinda deprecated so let's just return None prob